SMLHelper/Initializer.cs

```diff
@@ -90,3 +90,6 @@ public static void PostPatch()
         }
     }
 }
+
+// fix for C# 9.0 (for pre-5.0 .NET) for init-only properties and records
+namespace System.Runtime.CompilerServices { internal static class IsExternalInit {} }
```

Comment:
Yeah, it's exactly "sweep it under the rug", didn't find a better place for it. `Initializer.cs` is not quite an example of good code either, but yeah, I can create a new file for it.

Reply:
> `Initializer.cs` is not quite an example of good code either

Whilst I can empathise with this feeling, two wrongs don't make a right. Just because other code is sloppy doesn't mean you should also write sloppy code! Wherever we touch the codebase, we should aim to leave it better than we found it :)

Reply:
Yeah, yeah, I know about boyscouts ;)